modules/openapi-generator/src/main/resources/php-symfony/model_generic.mustache

```diff
@@ -21,11 +21,11 @@ class {{classname}} {{#parentSchema}}extends {{{parent}}} {{/parentSchema}}
     /**
      * Gets {{name}}.
      *
-     * @return {{{vendorExtensions.x-comment-type}}}
+     * @return {{{vendorExtensions.x-comment-type-for-getter}}}
      */
-    public function {{getter}}(){{#vendorExtensions.x-parameter-type}}: {{vendorExtensions.x-parameter-type}}{{/vendorExtensions.x-parameter-type}}
+    public function {{getter}}(){{#vendorExtensions.x-parameter-type-for-getter}}: {{vendorExtensions.x-parameter-type-for-getter}}{{/vendorExtensions.x-parameter-type-for-getter}}
     {
-        return $this->{{name}};
+        return {{#defaultValue}}$this->{{name}} ?? {{{defaultValue}}}{{/defaultValue}}{{^defaultValue}}$this->{{name}}{{/defaultValue}};
```

Comment:
@reznikartem thanks for the PR. Should the default value be assigned in the constructor instead?

If we do this in the getter, that means we can't set a property to null (assuming the property is nullable)

Reply:
Hi, @wing328
Thank you for answer. You are right, i`m missed this moment. Revert type-hinting in second commit to initial state. And move default value of properties into property initializator.

__constructor bad place for default values because of it`s not running when object created by jms serializer, from json string. 

In constructor i`m added checks with array_key_exist. Now default values will be present in objects, and they can be replaced with null for nullable properties, if this null value presents in $data array. If key not present in $data array, then default value will be available from getter.

Reply:
ok. let's give it a try.